public/locales/en/translation.json

```diff
@@ -77,8 +77,24 @@
     "select_pattern_placeholder": "Select pattern",
     "select_operations_placeholder": "Select",
     "original_data": "Original data",
+    "full_number": "(Full number)",
```

Comment:
according to https://metakeen.atlassian.net/browse/FEE-777 this should be removed